exercises/module-8-headless-apis/import_web_content_articles_batch.json

```diff
@@ -21,7 +21,7 @@
                         "title": "Eyewear glasses on Shelves"
                     }
                 },
-                "name": "Image"
+                "name": "image"
             }
         ],
         "contentStructureId": [content-structure-id],
```

Comment:
One of the participants mentioned that it would be nicer to have curly brace characters instead of brackets around the replaceable IDs (like {content-structure-id} instead of [content-structure-id]. What do you think @Claudioss23

Reply:
Yeah, I guess using curly braces is visually better than using brackets. I'll add the changes.

Should I also send the changes to the other files in this PR?

Reply:
please go ahead with changes in this PR